apps/client/src/features/rundown/event-block/EventBlock.utils.ts

```diff
@@ -39,3 +40,29 @@ export function formatOverlap(timeStart: number, previousStart?: number, previou
   const overlapString = formatDuration(Math.abs(timeFromPrevious), false);
   return `${timeFromPrevious < 0 ? 'Overlap' : 'Gap'} ${overlapString}`;
 }
+
+export function calculateAccumulatedGap(
+  accGap: MaybeNumber,
+  currentEvent: PlayableEvent,
+  previousEvent?: PlayableEvent,
+): MaybeNumber {
+  if (previousEvent === undefined) return null;
```

Comment:
suggestions: 

- we likely prefer spelled out variable names like `accumulatedGap` rather than `accGap`. Short names is the compilers job

- we would likely want some comments in this function to clarify what are the covered cases ie

`// check if there is an event before`
`// check if there is a gap from the previous event`
`// check if event is the day after`